flow_vectors[] parameters
uint 8 height
uint 8 width
